SolutionArchitecture.md

```diff
@@ -0,0 +1,101 @@
+### **Solution Architecture Walkthrough**
+
+![image](/images/readMe/image4.png "Inserting image...")
+
+Call and text recordings are uploaded to the web application. Those
+recordings are stored in a storage account and processed and have meta
+data applied to them using a set of Azure functions. Once those are
+processed, they leave the storage account and are pushed to the search
+index. In parallel, an Azure function runs to make a call to the Azure
+OpenAI Service to generate summarizations of the phone calls and other
+key conversation points. These results are surfaced in the app service
+where users can view the complete collections of calls with the
+summaries, customer satisfaction score, and keyphrases extracted from
+the conversation.
+
+### **Resource Group Walkthrough**
+
+After deploying our template, you should have a resource group that
+contains the following resources:
+
+-   Application Insights - this resource gives details on application
+    usage
+
+-   App Service Plan - there will be two of these resources in you resource group. These resources define a set of compute resources for a web app to run. You should not have to make any changes to these resources after deployment.
```

Comment:
This is an example of a really long line where many other descriptions are broken by hard carriage returns. Need to review so they are all consistent.